data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -101,16 +101,16 @@ done
 # plane.  We are using iptables instead of dnsmasq's dhcp-host because
 # DHCPv6 wants to use DUID's instead of mac addresses.
 {{if .PlatformData.BareMetal.ProvisioningDHCPAllowList}}
-$IPTABLES -t raw -N DHCP
-$IPTABLES -t raw -A PREROUTING -p udp --dport 67 -j DHCP
-$IPTABLES -t raw -A PREROUTING -p udp --dport 547 -j DHCP
+$IPTABLES -t raw -N DHCP || true
+$IPTABLES -t raw -A PREROUTING -p udp --dport 67 -j DHCP || true
+$IPTABLES -t raw -A PREROUTING -p udp --dport 547 -j DHCP || true
```

Comment:
I don't think unconditionally ignoring errors here is the most robust approach, how about we use `--check` to add a conditional, which will mean we don't attempt to append the rules when they already exist?

(See https://github.com/openshift-metal3/dev-scripts/pull/836 for an example of this, it's the approach used in dev-scripts prior to moving to ansible for firewall setup)

Reply:
Actually we also do the same on line 92 which is why I've not seen errors on restart before (the prerouting rules were added recently)